utils/apiPerformance.cjs

```diff
@@ -78,7 +78,8 @@ const trigger = function(gatewayUrl, username, password, team_name, project_name
         method: 'POST',
         headers: {
             'Content-Type': 'application/json'
-        }
+        },
+        rejectUnauthorized: false
```

Comment:
why is this flag used?

Reply:
was getting self signed certificate in certificate chain error when ever we try to trigger Unum instance , so we used it